recipe/meta.yaml

```diff
@@ -39,6 +39,7 @@ test:
     - dask
   commands:
     - pip check
+    - dask --help
```

Comment:
This was a test that was included when generating with `greyskull pypi dask`; thought it might be useful to include